File: sdk/samples/WPFSamples/WizardSample/visualbasic/wizardreturneventargs.vb
Imports System

Public Class WizardReturnEventArgs

    Public Sub New(ByVal result As WizardResult, ByVal data As Object)
        Me._result = result
        Me._data = data
    End Sub

    Public ReadOnly Property Data() As Object
        Get
            Return Me._data
        End Get
    End Property

    Public ReadOnly Property Result() As WizardResult
        Get
            Return Me._result
        End Get
    End Property

    Private _data As Object
    Private _result As WizardResult

End Class

File: sdk/samples/WPFSamples/WizardSample/csharp/wizardreturneventargs.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WizardSample
{
    public class WizardReturnEventArgs
    {
        WizardResult result;
        object data;

        public WizardReturnEventArgs(WizardResult result, object data)
        {
            this.result = result;
            this.data = data;
        }

        public WizardResult Result
        {
            get { return this.result; }
        }

        public object Data
        {
            get { return this.data; }
        }
    }
}
